src/components/ArticlesPage/ArticlesPage.jsx

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+
+function ArticlesPage() {
+    const [articles, setArticles] = useState([]);
+
+    function getAllArticles() {
+        return axios
+            .get("https://northcoders-news-c5lb.onrender.com/api/articles")
+            .then((response) => {
+                return response.data.articles;
+            });
+    };
+
+    useEffect(() => {
+        getAllArticles().then((returnedArticles) => {
+            setArticles(returnedArticles);
+        });
+    }, []);
```

Comment:
This page could really do with some sort of loading state - when I booted the page up this morning the server took about 15 seconds to spin up - in that meanwhile you want to make it clear that it's loading so that it's not unclear if the site it working or not